components/DropdownFilter.js

```diff
@@ -121,89 +121,69 @@ const Chevron = () => (
   </ChevronIcon>
 )
 
-class DropdownFilter extends React.Component {
-  constructor(props) {
-    super(props)
-    this.state = {
-      hoverArrow: false,
-      drop: false,
-      selected: props.selected,
-    }
-  }
+const DropdownFilter = ({ selected, name, options, updateTag }) => {
+  const [drop, setDrop] = useState(false)
 
-  toggleDrop() {
-    const { drop } = this.state
-    this.setState({ drop: !drop })
-  }
+  const toggleDrop = () => setDrop(!drop)
 
-  isSelected(tag) {
+  const isSelected = tag => {
```

Comment:
The name of the function could be changed to `getSelected` or something like that.